packages/devextreme/js/__internal/core/widget/dom_component.ts

```diff
@@ -86,6 +87,8 @@ class DOMComponent<
   }
 
   ctor(element: Element, options: TProperties): void {
+    waitUntilFirstWidgetCreating.resolve();
```

Comment:
As I've written above - since it's a promise, not direct callback call, it looks like `Created`, not `Creating`

So, shouldn't it be moved to the end of ctor body? Maybe, it should be called in the leaf child, so that it would be truly marks a componetn initialization is over?